dex/src/main/scala/com/wavesplatform/dex/market/CreateExchangeTransactionActor.scala

```diff
@@ -0,0 +1,48 @@
+package com.wavesplatform.dex.market
+
+import akka.actor.{Actor, Props}
+import com.wavesplatform.account.Address
+import com.wavesplatform.dex.market.CreateExchangeTransactionActor.OrderExecutedObserved
+import com.wavesplatform.dex.model.Events.{ExchangeTransactionCreated, OrderExecuted}
+import com.wavesplatform.dex.model.ExchangeTransactionCreator.CreateTransaction
+import com.wavesplatform.utils.ScorexLogging
+import play.api.libs.json.Json
+
+import scala.collection.mutable
+
+class CreateExchangeTransactionActor(createTransaction: CreateTransaction) extends Actor with ScorexLogging {
```

Comment:
Can you please add some brief explanation for the main purpose of this actor?
I mean the idea about awaiting confirmation from both address actors (or one) can be wrote down here in a short doc

Reply:
Done